dialog-el.html

```diff
@@ -355,7 +357,9 @@
       }.bind(this));
     }
 
-    proto.close = function() {
+    proto.close = function(e) {
+      // Allow persistent dialogs if the element clicked on has an open-dialog attribute.
+      if (e && e.target && e.target.attributes["open-dialog"]) return;
```

Comment:
The idea isn't to keep the dialog open all the time. I still want `esc` and clicking away to work. This is meant to fix a problem where I can't switch immediately from one dialog to another. I have to click away from the dialog first before opening another and that hurts usability. Adding the `open-dialog` attribute to the dialog needing to avoid a close seemed to work, but if you have a better solution I'm open to making changes.